Client/Clique/Common/Services/ProfileService.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  ProfileService.swift
+//  Clique
+//
+//  Created by Infinum on 29.11.2022..
+//
+
+import UIKit
+import Alamofire
+
+final class ProfileService {
+    func getUser(completion: @escaping(Result<UserProfile, Error>) -> Void) {
+        AF.request(Constants.Service.profileGetUserURL,
+                   method: .get,
+                   headers: Constants.Service.authorizationHeader())
+            .validate(statusCode: 200..<300)
+            .responseDecodable(of: UserProfile.self) {
+                dataResponse in
+                switch dataResponse.result {
+                case .success(let user):
+                    completion(.success(user))
+                case .failure(let error):
+                    completion(.failure(error))
+                }
+            }
+        }
+    
+    func updateUser(
+        with entries: UserProfileUpdateData,
+        completionHandler: @escaping(Bool)->()) {
```

Comment:
Please use `Result<Void, Error>` as a type inside the completion as you did in other service calls. I've mentioned that in #12 , so please take a look for the reference.